src/resolvers/types/deployment/index.js

```diff
@@ -134,6 +134,10 @@ export function deploymentCapabilities(parent, args, ctx) {
     {
       key: "canUpdateVariables",
       value: "deployment.variables.update"
+    },
+    {
+      key: "canUpdateUserRole",
+      value: "deployment.config.update"
```

Comment:
@schnie think we should add a new, specific role for this? `DEPLOYMENT_EDITOR` currently has the ability to `deployment.config.update`. Should this be limited to `DEPLOYMENT_ADMIN`?
```suggestion
      value: "deployment.userRoles.update"
```